WordPress/gutenberg
# Add aria labels to the focal point picker component

## What?

Previously, when using a screen reader, the focal point picker input controls would only announced "left" and "top" without context.
Before:
value, left, stepper
value, top, stepper

This PR adds an aria label to each input control. The visible text label is unchanged.
After:
value, focal point left position, stepper
value, focal point top position, stepper

Closes
https://github.com/WordPress/gutenberg/issues/50991

## Why?

The lack of context made the control confusing.

## How?

This PR adds an aria label to each input control. The visible text label is unchanged.

## Testing Instructions
Activate your screen reader. I tested with VoiceOver on mac.
In the block editor, add a cover block with an image.
Navigate to the block settings sidebar, and tab through the settings tab.
Confirm that the aria labels for the focal point picker are announced.
